x-pack/legacy/plugins/apm/server/lib/service_map/initialize_service_maps.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Server } from 'hapi';
+// @ts-ignore
+import { TaskManager, RunContext } from '../legacy/plugins/task_manager';
+import { runServiceMapTask } from './run_service_map_task';
+import { setupRequiredScripts } from './setup_required_scripts';
+
+export async function initializeServiceMaps(server: Server) {
+  const config = server.config();
+
+  setupRequiredScripts(server).catch(error => {
+    server.log(
+      'error',
+      'Unable to set up required scripts for APM Service Maps:'
+    );
+    server.log('error', error);
+  });
```

Comment:
Is it the intention to continue initialization if `setupRequiredScripts` fails?

Reply:
that was the idea, the result will be some error messages in kibana logs & service maps UI not displaying any nodes

Reply:
How's that useful? What about just terminating init as soon as something goes wrong?